internal/adapters/libparodus/libparodus.go

```diff
@@ -58,12 +58,6 @@ var _ Option = optionFunc(nil)
 
 // New creates a new Service with the given options.
 func New(url string, pubsub *pubsub.PubSub, opts ...Option) (*Adapter, error) {
-	defaults := []Option{
-		KeepaliveInterval(30 * time.Second),
-		ReceiveTimeout(1 * time.Second),
-		SendTimeout(1 * time.Second),
-	}
-
```

Comment:
removing to keep things consistent (where our defaults are set), since we already set defaults in `cmd/xmidt-agent/config.go`